pkg/connector/userinfo.go

```diff
@@ -25,10 +26,24 @@ func (m *MetaClient) wrapUserInfo(info types.UserInfo) *bridgev2.UserInfo {
 		identifiers = append(identifiers, fmt.Sprintf("instagram:%s", info.GetUsername()))
 	}
 
+	name := info.GetName()
+	if name == "" {
+		resp, err := m.Client.ExecuteTasks(&socket.GetContactsFullTask{
+			ContactID: info.GetFBID(),
```

Comment:
This is the wrong place, it should go in GetUserInfo and only run if `ghost.Name` is empty (otherwise just return nil)

Reply:
Hmm, I don't think anyone has reported `Unknown user`s, they're all completely unnamed: #107

PLAT-26900 is also all `@facebookgo_...` names